script/doc-only-change.js

```diff
@@ -0,0 +1,58 @@
+const args = require('minimist')(process.argv.slice(2))
+const octokit = require('@octokit/rest')()
+const path = require('path')
+
+const SOURCE_ROOT = path.normalize(path.dirname(__dirname))
+
+async function checkIfDocOnlyChange () {
+  if (args.prNumber || args.prBranch || args.prURL) {
```

Comment:
Hm... I don't think you need the PR number to get the diff. We should be able to use `git merge-base` to figure out the root of the branch, e.g.

```
$ git merge-base HEAD origin/master
```

should give you the base commit of the branch. Then you can do the diff from that base to `HEAD`.

Reply:
@nornagon will that work for fork PRs?

Reply:
Also, I don't think we know in CI which branch a PR is targeting

Reply:
I've done this dance before for https://circleci.com/gh/electron/archaeologist you need the PR number to get the `base` of the PR.  We don't know if it's targeting master or another branch